types.d.ts

```diff
@@ -100,6 +109,11 @@ interface TocList extends Element {
   children: TocListItem[];
 }
 
+interface Variables {
+  user: Record<string, string>;
+  defaults: { name: string; default: string }[];
+}
```

Comment:
Ahh, and this is the same structure we use for variables today, right? Convenient.